cmd/chasquid-util/chasquid-util.go

```diff
@@ -269,3 +271,56 @@ func domaininfoRemove() {
 		Fatalf("Error removing file: %v", err)
 	}
 }
+
+// chasquid-util alias-add <source> <target>
+func aliasAdd() {
+	source := args["<source>"].(string)
+	target := args["<target>"].(string)
+
+	sourceUser, sourceDomain := envelope.Split(source)
```

Comment:
Can you rename the variables to be just "user, domain", for simplicity? I think it's still readable given the context.
And check the domain is there, just in case?

https://github.com/albertito/chasquid/blob/da975fce7ceffe6ff97ffd556c937bff4c66f3db/cmd/chasquid-util/chasquid-util.go#L94-L97

Reply:
I thought about checking for presence of `domain` as well but the current domain is added to the target address if it's omitted, so it's probably valid to create aliases without it?
https://github.com/albertito/chasquid/blob/master/internal/aliases/aliases.go#L318-L322

Reply:
You are right that within the content of the aliases file, the source doesn't take a domain, and it's optional in the target.

But here, the source domain is used to identify which aliases file to use, so I don't see how it can be optional?

Reply:
Oh, I confused that, sry.. will add a check